packages/harpstrata/src/harpstrata/IsActive/IsActiveComplex/getIsActiveComplex/index.ts

```diff
@@ -1,13 +1,11 @@
-import type { IsActiveComplex } from '../types'
 import type { IsActiveProps } from '../../types'
-import { getIsActiveMatrix } from '../../IsActiveMatrix'
+import { ActiveIdsPair } from '../../ActiveIds/types'
```

Comment:
This isn't the right way to do this now is it.

Reply:
But this function needs to be deleted and the getActiveIdsPair function used directly by this functions caller instead.

Reply:
Done